test/component/QueueSpec.scala

```diff
@@ -98,4 +117,41 @@ class QueueSpec extends FeatureSpec
       contentAsString(unpulledResult) shouldBe xmlBody.toString()
     }
   }
+
```

Comment:
How about `"Post, pull messages and then get a list of all previously pulled messages from the queue"` rather than `"Post, pull messages and then get a list of all pulled messages from the queue"` to be more explicit? and same for following text that talks about `pulled`

Reply:
done